app/models/bulk_tax_return_update.rb

```diff
@@ -40,8 +42,8 @@ def data_stored_appropriately
 
   def updates
     updates = {}
-    if status.present?
-      updates["status"] = TaxReturnStatusHelper.status_translation(status)
+    if state.present?
+      updates["status"] = TaxReturnStatusHelper.status_translation(state)
```

Comment:
i _think_ this means that for any notifications that contain old btru's(that have a status but not a state), it'll look on the notifications tab like they didn't update the status/state

Reply:
hmm... you are probably correct. i think i need to go through the bulk tax return updates part of things and make sure we read from status OR state until we do the backfill, then push another PR to only look at state once the state is done.

Tbh, this review is making me remember that the bulk updates area needs more attention in this update. thanks!